spec/lib/root_route_constraint_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'spec_helper'
+
+describe RootRouteConstraint do
+  describe '#matches' do
+    it 'return true if only one conference is live' do
+      current = Conference.where('end_date >= ?', Date.current).reorder(start_date: :asc)
```

Comment:
You are not supposed to set this in the test, you are supposed to create a test environment with all the necessary resources you need in order to check if what you are expecting to happen, is actually happening.
In this case you are testing
1. if a conference exists, and the conference has the attributes we are interested in, then it matches
2. if a conference does not exist, then it does not match
3. if a conference exists but it does not have the necessary attributes, then it does not match